convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -36,19 +37,35 @@ def run(self):
 
         logger.task("Convert: List third-party packages")
         third_party_pkgs = pkghandler.get_third_party_pkgs()
+        third_party_package_skip = os.environ.get("CONVERT2RHEL_THIRD_PARTY_PACKAGE_CHECK_SKIP", None)
         if third_party_pkgs:
             pkg_list = pkghandler.format_pkg_info(third_party_pkgs)
-            message = (
+            warning_message = (
                 "Only packages signed by %s are to be"
                 " replaced. Red Hat support won't be provided"
                 " for the following third party packages:\n%s" % (system_info.name, ", ".join(pkg_list))
             )
-            logger.warning(message)
-            self.add_message(
-                level="WARNING",
-                id="THIRD_PARTY_PACKAGE_DETECTED",
-                message=message,
+
+            if not third_party_package_skip:
+                logger.warning(warning_message)
```

Comment:
This `logger.warning()` should not be here.  When we `set_result( [OVERRIDABLE])` then it counts as an error and the framework itself logs it. (Sorry, I may have not noticed that in a previous review of htis change.)